pkg/apiserver/metrics/metrics.go

```diff
@@ -42,7 +42,7 @@ var (
 			// Use buckets ranging from 125 ms to 8 seconds.
 			Buckets: prometheus.ExponentialBuckets(125000, 2.0, 7),
 		},
-		[]string{"verb", "resource"},
+		[]string{"verb", "resource", "client"},
```

Comment:
Yeah, @gmarek, would you be opposed to putting this in the summary instead?

GKE consumes this programmatically to ensure the health of our masters. We could adapt it or stop using it if needed since it isn't critical, but if using the summary would work for you that'd be easiest for me.

Also, you should be careful about how much extra memory this takes up - the number of clients hitting the apiserver is potentially unbounded, and can be quite high even in normal operation. I believe resistance from Red Hat is one of the main reasons we didn't include client in this metric originally.